cmd/kube-scheduler/app/server.go

```diff
@@ -234,6 +237,12 @@ func Run(ctx context.Context, cc *schedulerserverconfig.CompletedConfig, sched *
 			OnStoppedLeading: func() {
 				select {
 				case <-ctx.Done():
+					if cc.SecureServing != nil {
+						<-listenerStoppedCh
+						logger.V(1).Info("[graceful-termination] HTTP Server has stopped listening")
```

Comment:
```suggestion
						logger.Info("[graceful-termination] healthz server has stopped listening")
```

let's keep the log level same as above.